### ISSUE TITLE
Contact Form Submission - Mistake (Silver - Custom Comparators and Coordinate Compression)

### ISSUE THREAD

**Issue by maggieliu05** [cpinitiative/usaco-guide#4992]
Someone submitted the contact form!

**URL**: https://usaco.guide/silver/sorting-custom#comparators
**Module**: Silver - Custom Comparators and Coordinate Compression
**Topic**: Mistake
**Message**: 
sorting by multiple criteria, it should be "width != y.width"

**Comment by bqi343** [comment]
@SansPapyrus683 can you fix? and add a test case, since this section has been wrong multiple times ...

**Comment by SansPapyrus683** [comment]
uh in china so `git push` like doesn't work for some reason
will get on this when i get back

**Comment by ryanchou-dev** [comment]
> @SansPapyrus683 can you fix? and add a test case to check, since this section has been wrong multiple times ...

just to clarify, a test case within the code snippet?

**Comment by bqi343** [comment]
@ryanchou-dev yes, the codesnip should be able to run on its own (like the ones earlier in the module with main functions) and show the difference between the previous comparator and the new one.
